packages/bpk-foundations-common/base/colors/button.json

```diff
@@ -0,0 +1,190 @@
+{
+    "imports": ["./aliases.json"],
+    "global": {
+      "type": "color",
+      "category": "button-colors"
+    },
+    "props": {
+      "PRIVATE_BUTTON_DISABLED_BACKGROUND_DAY": {
```

Comment:
where a private colour is referring to a token, rather than being completely custom can we refer to the token instead? that way if we update for example the `corePrimary` colour we won't have to manually update the hex value where it's used in buttons

Reply:
Those colours are not referring to the 'real colours'. So the buttons would look wonky if we did that. Is that ok?

Reply:
The colour hexes referenced now are the 'final' new colours

Reply:
ah yes you're right! we should fix that after the fact (also for chips)